internal/xfn/function_runner.go

```diff
@@ -229,7 +229,7 @@ func (r *PackagedFunctionRunner) getClientConn(ctx context.Context, name string)
 		is[i] = r.interceptors[i].CreateInterceptor(name, active.Spec.Package)
 	}
 
-	conn, err := grpc.DialContext(ctx, active.Status.Endpoint,
+	conn, err := grpc.DialContext(ctx, active.Status.Endpoint, //nolint:staticcheck // Figure out how to replace with grpc.NewClient and still pass the dialFunctionTimeout.
```

Comment:
I spent some time trying to fix this but couldn't figure out a way to do it properly.